src/tabs/TabPane.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2021 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+
+export type TabPaneProps = {
+    id: string;
+};
+
+export class TabPane extends React.PureComponent<TabPaneProps> {
```

Comment:
Do we really need this component? we are not doing anything extra here other than wrapping child component in div tag

Reply:
We will need this component to ensure no other children other than TabPane gets displayed in tab.